### DISCUSSION TITLE
Controlling the backport title for the backport command (_not_ action)?

### DISCUSSION THREAD

**Discussion by geekosaur** [Mergifyio/mergify#5127] [Q&A]
I have been asked by a developer on our project if it is possible to rearrange the titles of created backports. We use the `@mergifyio backport` command rather than doing it automatically via merge rules.

It looks to me like, if I use the backport action in a merge rule, I have control over the PR title, but I don't see a way to do this when using the command and it doesn't look like I can specify a template separately from the action. Is this correct, or is there a way to do this?

**Comment by DouglasBlackwood** [comment]
Hi! You should be able to do this using the [default section](https://docs.mergify.com/configuration/file-format/#defaults). This section allows you to set configuration values for actions executed by pull request rules and **commands**.
